blog/zh/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md

```diff
@@ -1,50 +1,56 @@
 ---
-title: "GCP、AWS 和 Azure ARM 架构服务器性能测试对比"
+title: "GCP、AWS、Azure 和 Oracle ARM 架构服务器性能测试对比"
 authors:
   - name: "赵士瑞"
     title: "Author"
-    url: "https://github.com/soulbird"
+    url: "soulbird - Overview"
     image_url: "https://github.com/soulbird.png"
   - name: "苏钰"
     title: "Technical Writer"
-    url: "https://github.com/SylviaBABY"
+    url: "SylviaBABY - Overview"
     image_url: "https://avatars.githubusercontent.com/u/39793568?v=4"
 keywords: 
 - API 网关
 - ARM
 - Azure
 - AWS
+- Oracle
 - Google
 - Apache APISIX
-description: 本文使用 API 网关 Apache APISIX 来比较 GCP、AWS 和 Azure ARM 架构服务器在网络 IO 密集型场景下的性能。
+- description: 本文使用 API 网关 Apache APISIX 来比较 GCP、AWS 、Azure 和 Oracle ARM 架构服务器在网络 IO 密集型场景下的性能。
 tags: [Ecosystem]
```

Comment:
```suggestion
tags: [Ecosystem]
date: 2022-10-24
```

Reply:
Should we update the release date of this article? It's hard for people to see this article of ours without an update. Or you can edit Editors' Picks. 
please refer to: https://github.com/apache/apisix-website/blob/master/blog/en/config/picked-posts.json

Reply:
@juzhiyuan @hf400159 Which way should we choose? Could you make a decision?
Update the latest date, or put it into the picked posts?

Reply:
1. Update the `date` field to the latest date.
2. Put this post into https://github.com/apache/apisix-website/blob/master/blog/en/config/picked-posts.json#L2